server/src/main/java/com/adobe/testing/s3mock/S3MockApplication.java

```diff
@@ -77,9 +77,9 @@
 @EnableAutoConfiguration
 @ComponentScan
 public class S3MockApplication {
-  public static final int DEFAULT_HTTPS_PORT = 9191;
-  public static final int DEFAULT_HTTP_PORT = 9090;
-  public static final int RANDOM_PORT = 0;
+  private static final int DEFAULT_HTTPS_PORT = 9191;
+  private static final int DEFAULT_HTTP_PORT = 9090;
+  private static final int RANDOM_PORT = 0;
```

Comment:
Please leave them as public, they were meant to be used for programmatic starting... (using the properties below).